util/execdetails/execdetails.go

```diff
@@ -15,6 +15,8 @@ package execdetails
 
 import (
 	"fmt"
+	"github.com/pingcap/tipb/go-tipb"
```

Comment:
format this import

Reply:
It's my fault. 
All import problems have been fixed.